src/plugin.js

```diff
@@ -819,7 +819,7 @@ for (let i = 0; i < _readOnly.length; i++) {
  *          - False otherwise.
  */
 Flash.isSupported = function() {
-  return Flash.version()[0] >= 10;
+  return true; 
```

Comment:
Let's still use `Flash.version()[0] >= 10;` if we're on Internet Explorer and `true` otherwise. Right now this will break if someone's on IE without Flash installed.